lib/CrudService.js

```diff
@@ -37,6 +39,9 @@ const {
   PULLCMD,
 } = require('./consts')
 const { getStateQuery } = require('./CrudService.utils')
+const { JSONPath } = require('jsonpath-plus')
+const { pointerToPath } = require('./AdditionalCaster.utils')
```

Comment:
I'd probably move this function to a different `utils` file (or create a new one) since its scope is wider then the AdditionalCaster.

Probably, I'd also rename it to something like `getPathFromPointer` or something that makes it somehwat understandable.